easybuild/easyconfigs/r/R/R-4.5.1-gfbf-2025a.eb

```
@@ -0,0 +1,171 @@
+name = 'R'
+version = '4.5.1'
+
+homepage = 'https://www.r-project.org/'
+description = """R is a free software environment for statistical computing
+ and graphics."""
+
+toolchain = {'name': 'gfbf', 'version': '2025a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['b42a7921400386645b10105b91c68728787db5c4c83c9f6c30acdce632e1bb70']
+
+builddependencies = [
+    ('pkgconf',   '2.3.0'),
+    ('Autotools', '20240712'),
+]
+dependencies = [
+    ('X11', '20250521'),
+    ('Mesa', '25.1.3'),
+    ('libGLU', '9.0.3'),
+    ('cairo', '1.18.4'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.5'),
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.6.3'),
+    ('zlib', '1.3.1'),
+    ('SQLite', '3.47.2'),
+    ('PCRE2', '10.45'),
+    ('libpng', '1.6.48'),
+    ('libjpeg-turbo', '3.1.0'),
+    ('libwebp', '1.5.0'),
+    ('LibTIFF', '4.7.0'),
+    ('Java', '21', '', SYSTEM),
+    ('libgit2', '1.9.1'),
+    ('OpenSSL', '3', '', SYSTEM),
+    ('cURL', '8.11.1'),
+    ('Tk', '8.6.16'),
+    ('HarfBuzz', '11.2.1'),
+    ('FriBidi', '1.0.16'),
+]
+
+# Some R extensions (mclust, quantreg, waveslim for example) require the math library (-lm) to avoid undefined symbols.
+# Adding it to FLIBS makes sure it is present when needed.
+preconfigopts = 'export FLIBS="$FLIBS -lm" && '
+configopts = "--with-pic --enable-threads --enable-R-shlib --with-recommended-packages=no"
+
+
+exts_default_options = {
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+    'source_urls': ['https://cran.r-project.org/src/contrib/Archive/%(name)s',
+                    'https://cran.r-project.org/src/contrib/',
+                    'https://cran.freestatistics.org/src/contrib'],
```

I don't mind changes the order, but let's keep the formatting & comments:
```suggestion
    'source_urls': [
        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
        'https://cran.r-project.org/src/contrib/',  # current version of packages
        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
    ],

---

fixed in 23fcc1a + a381921